core/src/main/spark-2.1/scala/com/pingcap/tispark/TiSparkVersion.scala

```diff
@@ -0,0 +1,6 @@
+package com.pingcap.tispark
+
+object TiSparkVersion {
+  val version: String =
+    "Release Version: 1.1-SNAPSHOT\nGit Commit Hash: 22256357d90ee27fca93a22bcc56cf35943d2e5f\nGit Branch: support_spark_2.3.0\nUTC Build Time: 2018-09-21 06:22:11"
```

Comment:
Why is this here? Did you by mistake check it in?

Reply:
This SparkVersion is still useful when we call `select ti_version()`, I will modify the release version.